microprofile/server/src/main/java/io/helidon/microprofile/server/ServerCdiExtension.java

```diff
@@ -212,6 +212,9 @@ private void registerClasspathStaticContent(Config config) {
         cpBuilder.welcomeFileName(config.get("welcome")
                                           .asString()
                                           .orElse("index.html"));
+        config.get("tempFolder")
```

Comment:
As this is part of production code, the property name should follow convention.
Also directory is more commonly used.
So probably `tmp-dir` is the right configuration here

Reply:
fixed